src/map/clif.c

```diff
@@ -2445,10 +2457,18 @@ void clif_additem(struct map_session_data *sd, int n, int amount, int fail) {
 		p.bindOnEquipType = sd->status.inventory[n].bound && !itemdb->isstackable2(sd->inventory_data[n]) ? 2 : sd->inventory_data[n]->flag.bindonequip ? 1 : 0;
 #endif
 #if PACKETVER >= 20150226
-		for (i=0; i<5; i++){
-			p.option_data[i].index = 0;
-			p.option_data[i].value = 0;
-			p.option_data[i].param = 0;
+		for (i = 0; i < MAX_ITEM_OPTIONS; i++) {
```

Comment:
Block with adding options to pointer probably better move to separate function.
This will allow for plugins to change options what will be send to client